test/engine/IndexScanTest.cpp

```diff
@@ -488,3 +488,58 @@ TEST(IndexScan, computeResultReturnsEmptyGeneratorIfScanIsEmpty) {
     ADD_FAILURE() << "Generator should be empty" << std::endl;
   }
 }
+
+// _____________________________________________________________________________
+TEST(IndexScan, unlikelyToFitInCacheCalculatesSizeCorrectly) {
+  using ad_utility::MemorySize;
+  using V = Variable;
+  using I = TripleComponent::Iri;
+  auto qec = getQec("<x> <p> <s1>, <s2>. <x> <p2> <s1>.", true, false);
+  auto x = I::fromIriref("<x>");
+  auto p = I::fromIriref("<p>");
+  auto p2 = I::fromIriref("<p2>");
+
+  {
+    SparqlTripleSimple scanTriple{V{"?x"}, V{"?y"}, V{"?z"}};
+    IndexScan scan{qec, Permutation::Enum::POS, scanTriple};
```

Comment:
```suggestion
    IndexScan scan{qec, POS,{V{"?x"}, V{"?y"}, V{"?z"}}};
```

+ `using enum Permutation::Enum`